js/containers/portal-dashboard/popup-class-nav.tsx

```diff
@@ -0,0 +1,199 @@
+          HeaderIcon={SortIcon}
+          items={items}
+          trackEvent={trackEvent}
+          value={sortByMethod}
+          key={"feedback-sort"}
+        />
+      </div>
+    );
+  }
+
+  private renderViewListOptions() {
+    const { feedbackLevel, listViewMode, setListViewMode } = this.props;
+    const listByStudentClasses = `${css.toggle} ${css.listByStudents} ${listViewMode==="Student" ? css.selected : ""}`;
+    const listByQuestionsClasses = `${css.toggle} ${css.listByQuestions} ${listViewMode==="Question" ? css.selected : ""}  ${feedbackLevel === "Activity" ? css.disabled : ""}`;
+    return (
+      <div className={`${css.viewListOption} ${css.columnHeader}`}>View list by:
+        <button className={listByStudentClasses} data-cy="list-by-student-toggle" onClick={() => setListViewMode("Student")}>
+          <StudentViewIcon className={css.optionIcon} />
+        </button>
+        <button className={listByQuestionsClasses} data-cy="list-by-questions-toggle" onClick={() => setListViewMode("Question")} disabled={feedbackLevel === "Activity"}>
+          <QuestionViewIcon className={css.optionIcon} />
+        </button>
+      </div>
+    );
+  }
+
+  private renderSpotlightToggle() {
+    const spotLightContainerClasses = `${css.spotlightContainer} ${this.props.isSpotlightOn ? css.spotlightOn : ""}`;
+    return (
+      <div className={css.spotlightToggle} onClick={() => this.props.onShowDialog(true)} data-cy="spotlight-toggle">
+        <div className={spotLightContainerClasses}>
+          <SpotlightIcon className={css.spotlightIcon} />
+        </div>
+        <span>Spotlight selected</span>
+      </div>
+    );
+  }
+
+}
+
+function mapStateToProps(state: any, ownProps?: any) {
+  return (state: any, ownProps: any) => {
+    const { answers, currentQuestion, feedbackLevel } = ownProps;
+    if (feedbackLevel === "Question") {
+      const questionId = currentQuestion.get("id");
+      const questionFeedbacks = state.getIn(["feedback", "questionFeedbacks"]);
+      const feedbacksGiven = [];
+      questionFeedbacks.toArray().forEach((feedback: any) => {
+        if (feedback.get("questionId") === questionId && feedback.get("feedback").trim() !== "") {
+          feedbacksGiven.push(feedback);
+        }
+      });
+      const feedbackCount = feedbacksGiven.length || 0;
+      const studentAnswers = [];
+      answers.toArray().forEach((item: any) => {
+        item.forEach((ans: any) => {
+          if (ans.get("questionId") === questionId) {
+            studentAnswers.push(ans);
+          }
+        });
+      });
+      const numFeedbacksNeedingReview = studentAnswers.length - feedbackCount;
+      return { numFeedbacksNeedingReview };
+    } else {
+      const getFeedbacks: any = makeGetStudentFeedbacks();
+      const { numFeedbacksNeedingReview } = getFeedbacks(state, ownProps);
+      return { numFeedbacksNeedingReview };
+    }
+  };
+}
+
+export default connect(mapStateToProps)(PopupClassNav);
```

Comment:
I think we can get into a state where the student/question buttons are left in a disabled state based on previous user actions in the feedback mode.  I tried showing it in this gif.  First the buttons are enabled, but after entering one of the feedback modes, the question button is left disabled.
![button-disabled](https://user-images.githubusercontent.com/5126913/104769838-bf9b5780-5724-11eb-85af-31e2c19d51ec.gif)

Reply:
This has been fixed.